docs/reference/run.md

```diff
@@ -1385,30 +1385,71 @@ it will provide a named alias for the container being linked to.
 
 ### ENV (environment variables)
 
-When a new container is created, Docker will set the following environment
+On Linux, when a new container is created, Docker will set the following environment
```

Comment:
I see where you're going. It is `created` though as opposed to `start`. I essentially took what you had and will push an update shortly.